python/gui/qgscolorbutton.sip

```diff
@@ -1,329 +1,404 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/gui/qgscolorbutton.h                                             *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
 
-/** \ingroup gui
```

Comment:
I think the extra new line after this in the header cause loss of the class docstring